octopoes/octopoes/models/ooi/dns/records.py

```diff
@@ -155,3 +156,31 @@ class DNSPTRRecord(DNSRecord):
     @classmethod
     def format_reference_human_readable(cls, reference: Reference) -> str:
         return f"{reference.tokenized.address.address} -> {reference.tokenized.hostname.name}"
+
+
+class CAATAGS(Enum):
+    ISSUE = "issue"
+    ISSUEWILD = "issuewild"
+    IODEF = "iodef"
+    CONTACTEMAIL = "contactemail"
+    CONACTPHONE = "contactphone"
```

Comment:
issuevmc and issuemail are missing, see https://www.iana.org/assignments/pkix-parameters/pkix-parameters.xhtml#caa-properties

Reply:
I used an old RFC it seems. Added.